libtelemetry/src/test/java/com/mapbox/services/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -473,4 +473,37 @@ public void checksUnregisterReceiverWhenOptedOut() throws Exception {
 
     verify(mockedLocalBroadcastManager, times(1)).unregisterReceiver(eq(theEventReceiver));
   }
+
+  @Test
+  public void checksLocationPermissionSet() throws Exception {
+    Context mockedContext = mock(Context.class);
+    String aValidAccessToken = "validAccessToken";
+    EventsQueue mockedEventsQueue = mock(EventsQueue.class);
+    TelemetryClient mockedTelemetryClient = mock(TelemetryClient.class);
+    Callback mockedHttpCallback = mock(Callback.class);
+    SchedulerFlusher mockedSchedulerFlusher = mock(SchedulerFlusher.class);
+    Clock mockedClock = mock(Clock.class);
+    LocalBroadcastManager mockedLocalBroadcastManager = mock(LocalBroadcastManager.class);
+    MapboxTelemetry theMapboxTelemetry = new MapboxTelemetry(mockedContext, aValidAccessToken, mockedEventsQueue,
+      mockedTelemetryClient, mockedHttpCallback, mockedSchedulerFlusher, mockedClock, mockedLocalBroadcastManager);
+    theMapboxTelemetry.enable();
+
+    assertTrue(theMapboxTelemetry.optIn());
+  }
+
+  @Test
+  public void checksLocationPermissionNotSet() throws Exception {
```

Comment:
Here we're not testing the location permissions.
It actually returns `false` because telemetry is not enabled and `isTelemetryEnabled` is `false` https://github.com/mapbox/mapbox-events-android/blob/04732b8d5778e77d2a851632d0a722965e34a7ad/libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java#L200
We should revisit this test.